utils/ccloud_library.sh

```diff
@@ -1000,22 +1000,22 @@ function ccloud::destroy_ccloud_stack() {
   # Delete API keys associated to the service account
   ccloud api-key list --service-account $SERVICE_ACCOUNT_ID -o json | jq -r '.[].key' | xargs -I{} ccloud api-key delete {}
 
-  if [[ $KSQLDB_ENDPOINT != "" ]]; then
-    KSQLDB=$(ccloud ksql app list | grep $KSQLDB_NAME | awk '{print $1;}')
-    echo "KSQLDB: $KSQLDB"
-    ccloud ksql app delete $KSQLDB &>"$REDIRECT_TO"
+  if [[ "$KSQLDB_ENDPOINT" != "" ]]; then # This is just a quick check, if set there is a KSQLDB in this stack
+    local ksqldb_id=$(ccloud ksql app list -o json | jq -r 'map(select(.name == "'"$KSQLDB_NAME"'")) | .[].id')
```

Comment:
Do we know if these `jq` functions like map and select require a specific jq version?

Reply:
I do not, did you encounter an issue with the changes?

Reply:
Changes verified with 1.6, released Nov. 2018